instrumentation/README.md

```diff
@@ -73,6 +76,17 @@ The following methods are supported to manually activate and configure Auto Inst
      ```
      NODE_OPTIONS=-r /usr/lib/splunk-instrumentation/splunk-otel-js/node_modules/@splunk/otel/instrument
      ```
+   - `/etc/splunk/zeroconfig/dotnet.conf`:
+     ```
+     CORECLR_ENABLE_PROFILING=1
+     CORECLR_PROFILER={918728DD-259F-4A6A-AC2B-B85E1B658318}
+     CORECLR_PROFILER_PATH=/usr/lib/splunk-instrumentation/splunk-otel-dotnet/linux-x64/OpenTelemetry.AutoInstrumentation.Native.so
```

Comment:
Just to be sure that you are aware: since this is for deb/rpm packages `linux-x64` is correct, when using arm64 the path will change and also for Alpine.

Reply:
Yes, we can update the paths accordingly when `splunk-otel-dotnet` supports arm64 or when we need to support zero config for alpine.